README.md

```diff
@@ -22,6 +22,16 @@ typings search tape
 # Find an available definition (by name).
 typings search --name react
 
+# Install ambient typings (default from "DT", configurable through `defaultAmbientSource` in `.typingsrc`)
+typings install react --ambient --save
+
+# Install non-ambient typings (default from "npm" registry, configurable through `defaultSource` in `.typingsrc`)
+typings install popsicle --save
```

Comment:
This won't actually work, since `popsicle` is not in any registry. It's a scheme, installed with `typings install npm:popsicle`.